circle.yml

```diff
@@ -1,7 +1,17 @@
 machine:
+  environment:
+    PATH: "${PATH}:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin"
   node:
     version: 6.9
 
+dependencies:
+  override:
+    - yarn
```

Comment:
You may want to do `yarn install --frozen-lockfile`.

The `--frozen-lockfile` option forces `yarn` to ensure the `yarn.lock` file is valid given the project's `package.json` file.

The scenario where that flag is useful is if you, or someone else, where to `npm install` a new dependency, not using `yarn`, and then push the change to the repository. `--frozen-lockfile`, will fail the build early indicating the discrepancy between your `package.json` file, and your `yarn.lock` file.

Sadly, no documentation exists yet: yarnpkg/yarn#2478